modules/rostests/apitests/ntdll/NtFreeVirtualMemory.c

```diff
@@ -167,8 +167,48 @@ static void Test_NtFreeVirtualMemory(void)
     ok(Length == 2*PAGE_SIZE, "Length mismatch : 0x%08lx\n", (ULONG)Length);
     ok(Buffer2 == Buffer, "The buffer is not aligned to PAGE_SIZE.\n");
 }
-    
+
+static void Test_NtFreeVirtualMemory_Parameters(void)
+{
+    NTSTATUS Status;
+    ULONG FreeType;
+    int i;
+
+    // 4th parameter: "ULONG FreeType".
```

Comment:
Leftover?

Reply:
No, I meant it as a "header". Wrt possibly testing other parameters too.